Q/QD/build_tarballs.jl

```diff
@@ -0,0 +1,51 @@
+# We need to disable ccache in this build, because ccache doesn't support
+# Fortran 90 and modules would not be built otherwise.
+ENV["BINARYBUILDER_USE_CCACHE"] = "false"
+
+using BinaryBuilder
+
+name = "QD"
+version = v"2.3.22"
+
+# Collection of sources required to build SDPA-QD
+sources = [
+    "https://www.davidhbailey.com/dhbsoftware/qd-2.3.22.tar.gz" =>
+    "30c1ffe46b95a0e9fa91085949ee5fca85f97ff7b41cd5fe79f79bab730206d3",
+    "./bundled",
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/qd-2.3.22
+atomic_patch -p1 ../patches/static_constants.patch
+
+update_configure_scripts
+
+if [[ "${target}" == *-freebsd* ]]; then
+    # Regenerate the configure to be able to build the shared libraries
+    autoreconf -vi
+fi
+
+./configure --enable-shared --enable-fast-install=no --prefix=$prefix --host=$target --build=${MACHTYPE}
+make -j${nproc} module_ext=mod
+make install module_ext=mod
+
+install_license $WORKSPACE/srcdir/LBNL-BSD-License.docx
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms()
+
+# The products that we will ensure are always built
+products = [
+    LibraryProduct("libqd_f_main", :libqd_f_main),
+    LibraryProduct("libqdmod", :libqdmod),
```

Comment:
> I should say that ericphanson/SDPA_GMP_Builder:build_tarballs.jl@master works OK and uses QD, so somehow this isn't a problem there.

Also the [library in your SDPA-QD](https://github.com/ericphanson/QD_Builder/releases/download/v2.3.22/SDPA-QD.v2.3.22.x86_64-linux-gnu.tar.gz) has the symbol undefined:

```
% nm SDPA-QD.v2.3.22.x86_64-linux-gnu/lib/libqdmod.so | grep _ZN7dd_real8_ndigitsE
                 U _ZN7dd_real8_ndigitsE
```

> However, it never gets dlopen'd by Julia since SDPA-DD (and -QD and -GMP) do not expose a shared library, only a binary executable, so we have to shell out to call them...

Uhm, then for sake of our sanity what about not opening `libqdmod`?  It's not ideal, but if you don't need to directly call it that could be a work around.

```suggestion
    LibraryProduct("libqdmod", :libqdmod, dont_dlopen = true),
```

Reply:
Ah, I didn't know that was an option! Ok, sounds good